unity/UnityEmbedHost.Tests/BaseEmbeddingApiTests.cs

```diff
@@ -45,6 +46,49 @@ public void MonoObjectGetClassReturnsClass()
     //     }
     // }
 
+    class TestException : Exception
+    {
+        public TestException(string msg) : base(msg)
+        {
+        }
+    }
+
+    /*[Test] // Commented out because currently there is no way to get the GC Handle for the test assembly correctly
```

Comment:
Could you use an exception type defined in the class libraries to workaround this limitation?